src/herder/TransactionQueue.h

```diff
@@ -66,6 +66,7 @@ class TransactionQueue
         ADD_STATUS_DUPLICATE,
         ADD_STATUS_ERROR,
         ADD_STATUS_TRY_AGAIN_LATER,
+        ADD_STATUS_FILTERED,
```

Comment:
Looks like you need to fix `CommandHandler::testTx` that could crash. I'm not sure why it's done like this, it would be better to write the code there the same way than how `CommandHandler::tx` is written (and just use `TX_STATUS_STRING` instead of adding yet another way of converting codes to string)

Reply:
Good catch, I'll fix this. I'll make them work the same way because this is an obvious foot-gun.